crates/examples/src/readobj/elf.rs

```diff
@@ -611,6 +611,12 @@ fn print_notes<Elf: FileHeader>(
                                         p.flags(val, 0, FLAGS_GNU_PROPERTY_AARCH64_FEATURE_1);
                                     }
                                 }
+                                GNU_PROPERTY_AARCH64_FEATURE_PAUTH => {
+                                    if let Some(val) = property.data_u32(endian).print_err(p) {
```

Comment:
According to the ABI, this value "is at least two 64-bit words, the first being a platform identifier, and the second being a version number specific to the platform identified in the first word".